sonarqube/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 3.2.1 / 2024-04-26
+
+***Fixed***:
+
+* Declare the integration as  Python 2 compatible ([#17323](https://github.com/DataDog/integrations-core/pull/17323))
```

Comment:
```suggestion
* Declare the integration as Python 2 compatible ([#17323](https://github.com/DataDog/integrations-core/pull/17323))
```

Reply:
Sorry @brett0000FF I have to keep this consistent with the original PR.

Reply:
Gotcha 👍